.travis.yml

```diff
@@ -1,20 +1,19 @@
 ---
+dist: xenial
 language: ruby
-sudo: false
 cache: bundler
 rvm:
 - 2.4.5
+- 2.5.3
 addons:
-  postgresql: '9.5'
-env:
-  global:
-  - DATABASE_URL="postgresql://postgres:@localhost:5432/sources_test?encoding=utf8&pool=5&wait_timeout=5"
+  postgresql: '10'
 before_install:
-- gem update --system
+- 'echo ''gem: --no-ri --no-rdoc --no-document'' > ~/.gemrc'
 - gem install bundler
+- DATABASE_URL="postgresql://postgres:@localhost:5432/sources_api_test?encoding=utf8&pool=5&wait_timeout=5"
```

Comment:
Should this be in `env.global`?

Reply:
no?  I don't know what you mean

Reply:
The tests are failing when looking for a config/database.yml, I assume because there is no DATABASE_URL, possibly because this should be in the env global section rather than in before_install.

Reply:
Oh you mean in the section above?  No, it's not necessary...you only really need the env section if you plan on testing a matrix where the env changes actually emit multiple jobs.  See https://docs.travis-ci.com/user/build-matrix/

Reply:
This is how I set it up in all repos.  Not sure why this is different.  Let's discuss

Reply:
ok fixed